Update tutorial documents for dApp users. (#488)
klaytn/klaytn-docs: docs/dapp/tutorials/klaytn-online-toolkit.md

# Klaytn Online Toolkit <a id="klaytn-online-toolkit"></a>

> ## What is the Klaytn Online Toolkit?

Reviewer: You need to add <a> tag in every title.
```suggestion
## What is the Klaytn Online Toolkit? <a id="what-is-the-klaytn-online-toolkit"></a>
```

> * `Klaytn Online Toolkit` provides code examples and github page to help you to utilize the `Klaytn SDK(caver-js)` easily.

Reviewer: ```suggestion
* `Klaytn Online Toolkit` provides code examples to help you to utilize the `Klaytn SDK(caver-js)` easily. Also it provides a [demo page](https://klaytn.github.io/klaytn-online-toolkit/) for developers to use simple online tools.
```

> * `Klatyn SDK(caver-js)` is a JavasScript API library that allows developers to interact with a Klaytn node using a HTTP or Websocket connection.

Reviewer: ```suggestion
* `Klaytn SDK(caver-js)` is a JavaScript API library that allows developers to interact with a Klaytn node using a HTTP or Websocket connection.
```

> * You can just try out Klatyn's features without having to code.

Reviewer: ```suggestion
* You can just try out Klaytn's features without having to code.
```

> * `Klaytn SDK(caver-js)` is a JavaScript API library that allows developers to interact with a Klaytn node using a HTTP or Websocket connection.

Reviewer 2: ```suggestion
* `Klaytn SDK(caver-js)` is a JavaScript API library that allows developers to interact with a Klaytn node using an HTTP or Websocket connection.
```

> To help more people use the `Klaytn Online Toolkit`, We have prepared the ["Using Klaytn Online Toolkit"](https://medium.com/klaytn/using-klaytn-online-toolkit-1-multisig-60399a0b0278) series.

## Links

> Here are the links for `Klaytn Online Toolkit`. Feel free to it :)

Reviewer: ```suggestion
Here are the links for `Klaytn Online Toolkit`. Feel free to use it :)
```
